resources/home/dnanexus/Pan_Cancer_Sample_Report.Rmd

```diff
@@ -313,66 +319,67 @@ The table below lists URL links to different annotation databases. These are:
 
 
 ```{r url_links}
-fusion_abdriged_annotation$gene_fusion1 <- genes[seq_along(genes) %% 2 > 0]
-fusion_abdriged_annotation$gene_fusion2 <- genes[seq_along(genes) %% 2 == 0]
```

Comment:
Is using mod the best way to separate the two partners? Once one element of an array is wrong, all the rest of the elements will be wrong. How about using a data.frame/matrix where the a pair of partner genes can be clearly located?